### ISSUE TITLE
Allow landscape orientation on mobile phone (nice to have)

### ISSUE THREAD

**Issue by mschwendener** [bluesky-social/social-app#1680]
On my iPhone, when looking at feeds with pictures, I'd like to be able to turn the phone to lanscape mode to see the images bigger. 

(It's not blocked on phone level, it works in the browser for example.)

**Comment by chrismarquardt** [comment]
+1, Bluesky is the only social media app that doesn't allow landscape orientation for image viewing. This makes it especially hard when screenshots of text are involved.

**Comment by geir55** [comment]
Please fix

**Comment by ViaFerrata** [comment]
This is not available yet, right?
Would be great for tablet users.
Till then I'm stuck to using the browser website.

**Comment by mschwendener** [comment]
This is partly fixed with the latest update. 
But I need to click on the picture, which I then can see in landscape mode by turning the phone.

Are there plans to allow landscape mode for all of Bluesky = without the extra click?

**Comment by gaearon** [comment]
We’re not currently planning to support landscape for feeds themselves (at least not in short term anyway — though i haven’t seen feed apps commonly do that in general). But I think we can close this issue since it’s done for images.

**Comment by ViaFerrata** [comment]
> We’re not currently planning to support landscape for feeds themselves (at least not in short term anyway — though i haven’t seen feed apps commonly do that in general). But I think we can close this issue since it’s done for images.

X does it quite well.

**Comment by mschwendener** [comment]
> We’re not currently planning to support landscape for feeds themselves (at least not in short term anyway)

Thank you for the information, gaearon. 

Quite a few people hoping for this, mainly for iPad it seems. But maybe that's a separate subject.

**Comment by twilightscapes** [comment]
This is a problem for sites that link to videos, as the video is stuck in vertical orientation. The only way is to hit the browser icon and open in a browser. Most people will not do this, and it's an extra step, when they could just rotate their phone and enjoy a full screen version instead of being stuck permanently in vertical view. This has nothing to do with how Bluesky displays feeds, but how the webview forces external sites to be viewed in mobile view only.
